pkgs/development/python-modules/array-record/default.nix

```
@@ -1,27 +1,36 @@
 { lib
-, absl-py
 , buildPythonPackage
-, etils
+, pythonOlder
+, python
 , fetchPypi
+, absl-py
+, etils
 , importlib-resources
-, python
 , typing-extensions
 , zipp
 }:
 
 buildPythonPackage rec {
   pname = "array-record";
-  version = "0.4.1";
+  version = "0.5.0";
   format = "wheel";
 
-  disabled = python.pythonVersion != "3.10";
+  disabled = pythonOlder "3.9";
```

```suggestion
  # As of 2023-10-31, PyPI includes wheels for Python 3.9, 3.10, and 3.11.
  disabled = pythonOlder "3.9";
```
not sure what the best way to implement that version constraint is off the top of my head, but you get the idea

---

I just added the comment.